katas/content/single_qubit_gates/index.md

```diff
@@ -365,6 +365,65 @@ All the basic gates we will be covering in this kata are part of the <a href="ht
     "solutionPath": "./sign_flip_on_zero/solution.md"
 })
 
+@[section]({
+    "id": "identity_gate",
+    "title": "Identity Gate"
+})
+
+The identity gate is mostly here for completeness, at least for now. It will come in handy when dealing with multi-qubit systems and multi-qubit gates. It is represented by the identity matrix, and does not affect the state of the qubit.
+  
+<table>
+<tr>
+<th>Gate</th>
+<th>Matrix</th>
+<th>Ket-Bra</th>
+<th>Applying to $|\psi\rangle = \alpha|0\rangle + \beta|1\rangle$</th>
+<th>Q# Documentation</th>
+</tr>
+<tr>
+<td>$I$</td>
+<td>$\begin{bmatrix} 1 & 0 \\ 0 & 1 \end{bmatrix}$</td>
+<td>$|0\rangle\langle0| + |1\rangle\langle1|$</td>
+<td>$I|\psi\rangle = |\psi\rangle$</td>
+<td><a href=\"https://docs.microsoft.com/qsharp/api/qsharp/microsoft.quantum.intrinsic.i\">I</a></td>
```

Comment:
This link doesn't work for me, looking at other links within tables, we don't need to escape the double-quotes with backslashes to make them work as links

I'm wondering if we should remove the links to individual gates from this whole kata, since they all lead to classic QDK API pages that say explicitly that they are deprecated, and that's not the best user experience...

Reply:
That would be good. Those links were recently deprecated so now is a good time to remove them.

Reply:
Removed the "Q# Documentation" column from all of the tables in that Kata. 
Removed links to the old Intrinsic name space and DumpMachine documentation. Should that content be removed from the Kata as well?

Reply:
You mean the original kata, https://github.com/microsoft/QuantumKatas/blob/main/tutorials/SingleQubitGates/SingleQubitGates.ipynb? Technically the old Quantum Katas project is still running on the classic QDK, for which these links are accurate, so it makes sense to keep them there for now. The playground and the quantum.microsoft.com website both run on the modern QDK, for which these links are not necessarily accurate, that's why we want to remove them here.

A good place to remove more of those links would be the https://github.com/microsoft/qsharp/tree/main/katas/content/multi_qubit_systems tutorial, it follows a similar structure of gate description and has a couple of links to APIs of the gates that are deprecated as well.  The link to Controlled functor, though, is fine, since it leads to non-deprecated page in the main documentation